reviewers/attr_value_not_empty.go

```diff
@@ -26,6 +32,12 @@ func (a AttrValueNotEmpty) Review(path string, r io.Reader) ([]Fault, error) {
 			break
 		}
 
+		if rgx.Match(z.Raw()) {
+			b := rgx.ReplaceAll(z.Raw(), []byte{})
+			b = append(b, z.Buffered()...)
+			z = html.NewTokenizer(bytes.NewBuffer(b))
```

Comment:
@tatang26 could we do this replace at once before initializing z ?

Reply:
in here:
https://github.com/wawandco/milo/pull/28/files#diff-fc1fef914d1aca1d7f2273e250de9b1cR28

Reply:
Sure